src/commands/deploy/deploy.ts

```diff
@@ -54,7 +54,10 @@ export async function deploy(context: IActionContext, target?: vscode.Uri | Site
     const { node, isNewWebApp }: IDeployNode = await getDeployNode(deployContext, target, isTargetNewWebApp);
 
     if (node instanceof TrialAppTreeItem) {
-        return await deployTrialApp(deployContext, node);
+        await deployTrialApp(deployContext, node);
```

Comment:
Did you verify this works?  This seems like it's in the wrong order to me.  I figured deployTrialApp would be called last.

Reply:
It works, but I'm now realizing that I should call deployTrialApp last so that it actually works the first time they deploy.